package.json

```diff
@@ -68,9 +68,26 @@
     "ts-node": "^10.9.1",
     "ts-node-dev": "^2.0.0",
     "typescript": "^4.9.5",
-    "webpack": "^5.11.0"
+    "webpack": "^5.27.0"
```

Comment:
This dep can probably be removed, along with `babel-loader` which, is probably the reason this is here in the first place as it's a peer dep of it.

Reply:
Removing them breaks E2E tests 🤷‍♂️